core/sys/cc.h

```diff
@@ -110,6 +111,12 @@
 #define CC_INLINE
 #endif /* CC_CONF_INLINE */
 
+#ifdef CC_CONF_ALIGN_ATTR
+#define CC_ALIGN_ATTR CC_CONF_ALIGN_ATTR
+#else /* CC_CONF_INLINE */
+#define CC_ALIGN_ATTR
```

Comment:
If the requested alignment is a requirement and not a simple performance hint, this will silently generate a runtime bug.